modules/installation-mirroring-additional-images.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-restricted-networks-preparations.adoc
+
+[id="installation-mirroring-additional-images_{context}"]
+= Mirroring additional images
+
+When working in an isolated or restricted network, you may need to mirror additional must-gather images from an internal registry to a production registry. You can use the link:https://github.com/containers/skopeo[skopeo] command line tool to copy an image from one registry to another.
+
+.Procedure
+. Copy an image from a production registry to an isolated registry:
+[source,terminal]
++
+----
+skopeo copy docker://quay.io/openshift/origin-must-gather:latest
+docker://isolated.registry/myimage:v1.0
```

Comment:
Is this output from the above command? If so, we'll want it in a separate code block. We use the following syntax for doing example output from commands:

```
[source,terminal]
----
$ <command>
----
+
.Example output
+
[source,terminal]
----
<output>
----
```